md_rahaman/python/django/random_word_generator/apps/random_word_generator_app/templates/random_word_generator_app/index.html

```diff
@@ -0,0 +1,22 @@
+
+<html>
+	<head>
+		<title>Random World Generator</title>
+		<style type="text/css">
```

Comment:
There wasn't much css to do, so decided to do that, but i will not do it in future